lib/registration.pm

```diff
@@ -80,13 +80,18 @@ sub is_module {
 
 sub accept_addons_license {
     my (@scc_addons) = @_;
-
+    my @addons_with_license = "";
     # To check the current state of licenses in the product one can conduct
     # the following steps, e.g. for SLE15:
     #   isc co SUSE:SLE-15:GA 000product
     #   grep -l EULA SUSE:SLE-15:GA/000product/*.product | sed 's/.product//'
     # All shown products have a license that should be checked.
-    my @addons_with_license = qw(geo rt idu ids);
+    if (is_sle('<15')) {
+        @addons_with_license = qw(geo rt idu ids script);
+    }
+    else {
+        @addons_with_license = qw(geo rt idu ids);
+    }
```

Comment:
commonly the module is called "wsm" for "web and scripting module". Is there are reason why you need to use "script" here?

how about

```perl
my @addons_with_license = qw(geo rt idu ids);
push @addons_with_license, 'wsm' if is_sle('<15');
```


Can you comment why this seems to cause a problem now?

Reply:
This looks better. 
SLE-12-SP4 also provides modules, I think if I want to use it some day, there should be a way to do this. In fact, I already started using them during testing.

Reply:
Changes have been made. 'script' is changed to 'wsm' according to your advice. And I also add 'legacy' into it. Please correct me if the 'legacy' name should be something else. Otherwise, this PR should be ready to go.

Reply:
Changes have been made.